docs/vm-ee-guide.md

```diff
@@ -0,0 +1,267 @@
+# Onboard
+
+## Table of Contents
+
+<!-- toc -->
+- [VirtualMachines](#virtualmachines)
+    - [Labels](#labels)
+    - [Cloud Tags](#cloud-tags)
+    - [Sample VirtualMachine Output](#sample-virtualmachine-output)
+        - [AWS](#aws)
+        - [Azure](#azure)
+- [ExternalEntity](#externalentity)
+    - [Sample ExternalEntity Output](#sample-externalentity-output)
+        - [Azure](#azure-1)
+    - [Labels](#labels-1)
+    - [Cloud Tags](#cloud-tags-1)
+        - [Usage](#usage)
+<!-- /toc -->
+
+## VirtualMachines
+
+VirtualMachines are onboarded by specifying the matching criteria in `CloudEntitySelector` CR. The virtual machines from
+cloud are imported as `VirtualMachine` objects in `CloudEntitySelector` namespace. `VirtualMachine` object `Name` is a
+unique resource name within the kubernetes cluster. In Azure, ascii sum of the characters in VM Resource ID added as
+suffix to name of Virtual Machine and in AWS, `instance ID` of Virtual Machine is used as `Name`.
+
+### Labels
+
+Meaning of some label fields in `VirtualMachine` objects differ based on the cloud provider type.
+
+| Label        | Description                               | Azure                                    | AWS                          |
+|--------------|-------------------------------------------|------------------------------------------|------------------------------|
+| cloud-vm-uid | Unique identifier of VM                   | `vmId` field on cloud                    | `Instance ID` of VM on cloud |
```

Comment:
drop ' '. and rename as VM Instance ID. similarly for VPC it should be VPC ID.

Reply:
done